server/index.js

```diff
@@ -21,6 +21,16 @@ const app = express()
 
 const {preload, postload} = allMiddleware
 
+// The trust proxy flag tells the app to use https for links
+// and redirect urls if it sees indications that the request
+// passed through a proxy and was originally sent using https
+if ('TRUST_PROXY' in process.env) {
```

Comment:
Our convention elsewhere in the Library code is to fall back to an empty string for transformations rather than build a nested conditional. What do you think about writing the condition like this instead?
```suggestion
if ((process.env.TRUST_PROXY || '').toUpperCase() === 'TRUE') {
```